editor/header/index.js

```diff
@@ -59,22 +71,52 @@ function Header( { multiSelectedBlockUids, onRemove, onDeselect } ) {
 			aria-label={ __( 'Editor toolbar' ) }
 			className="editor-header"
 		>
-			<ModeSwitcher />
-			<SavedState />
-			<Tools />
+			<div className="editor-header__left">
+				<Inserter position="bottom right" />
+				<IconButton
+					className="editor-tools__undo"
```

Comment:
Shouldn't the namespace be the file component?

Reply:
yep sorry, copy/paste